static/js/dashboards-home.js

```diff
@@ -452,6 +451,19 @@ const dbgridOptions = {
 
 
 function displayDashboards(res, flag) {
+	let favorites = [];
+	let nonFavorites = [];
+	
+	for (let [key, value] of Object.entries(res)) {
+		if (favoriteDBsSet.has(key)) {
+			favorites.push([key, value]);
+		} else {
+			nonFavorites.push([key, value]);
+		}
+	}
+	
+	let resArray = [...favorites, ...nonFavorites];
+	res = Object.fromEntries(resArray);
```

Comment:
Previously, it looks like we were showing the dashboards in reverse alphabetical order. You should preserve this order for the favorites list and for the nonFavorites list. When I test this out, all the favorites are correctly at the top, but the elements of each list appear to be in a random order